arrow-optics-test/src/main/kotlin/arrow/test/generators/Generators.kt

```diff
@@ -1,123 +1,7 @@
 package arrow.test.generators
```

Comment:
Hi! Please, if `arrow-optics-test` is going to be published in OSS repository, replace `arrow.test` package by `arrow.optics.test` in order to avoid misunderstandings with other test artifacts. We'll do the same with  arrow-core-test` that is already published. Thanks!

Reply:
Will do!